packages/runtime/src/runtime/body.tsx

```diff
@@ -18,7 +19,7 @@ interface EnsembleBodyProps {
 }
 
 export const EnsembleBody: React.FC<EnsembleBodyProps> = ({ body, styles }) => {
-  const evaluatedStyles = useEvaluate({ styles });
+  const { values } = useRegisterBindings({ styles });
```

Comment:
why move to using `useRegisterBindings` hook?

Reply:
because in style if we passed classname, then we need to extract css from that classname, useEvaluate only evaluate it as js and resolve variable values

Reply:
@anserwaseem

Reply:
@anserwaseem  That is not needed, because if we include named and classname handling into useEvaluate, then it will become same expensive as useRegisterBindings,

and that's why I am only passing the styles into useRegisterBinding to avoid unnecessary load

Reply:
it will just fetch the styles from the theme, sometimes there are multiple level of expressions or variables assigned, I was in this situation some months ago, me and Evan spend much time to find a better solution and then we decided this, If you want to test, you can test in your side by placing variables into themes and also in screen yaml